src/ui/flutter_app/lib/l10n/intl_en.arb

```diff
@@ -272,25 +292,55 @@
     "@black": {
         "description": "Player 2"
     },
-    "loseReasonlessThanThree": " piece count is less than three.",
+    "loseReasonlessThanThree": "{loser} piece count is less than three.",
     "@loseReasonlessThanThree": {
-        "description": " piece count is less than three."
-    },
-    "loseReasonResign": " resign.",
+        "description": "Shows that the player lost because they had less than three pieces.",
+        "placeholders": {
+            "loser": {
+                "description": "The losing player",
+                "example": "Player 1"
+            }
+        }
+    },
+    "loseReasonResign": "{loser} resign.",
     "@loseReasonResign": {
-        "description": " resign."
-    },
-    "loseReasonNoWay": " is no way to go.",
+        "description": "Shows that the player lost because they had resigned.",
+        "placeholders": {
+            "loser": {
+                "description": "The losing player",
+                "example": "Player 1"
+            }
+        }
+    },
+    "loseReasonNoWay": "{loser} is no way to go.",
     "@loseReasonNoWay": {
-        "description": " is no way to go."
-    },
-    "loseReasonBoardIsFull": "The board is full, and no way to go.",
+        "description": "Shows that the player lost because they had no way to go.",
+        "placeholders": {
+            "loser": {
+                "description": "The losing player",
+                "example": "Player 1"
+            }
+        }
+    },
+    "loseReasonBoardIsFull": "{loser} The board is full, and no way to go.",
     "@loseReasonBoardIsFull": {
-        "description": "The board is full, no way to go."
-    },
-    "loseReasonTimeOver": "Time Over",
+        "description": "The board is full, no way to go.",
+        "placeholders": {
+            "loser": {
+                "description": "The losing player",
+                "example": "Player 1"
+            }
+        }
+    },
+    "loseReasonTimeOver": "{loser} Time Over",
```

Comment:
This is not needed.

Reply:
it is till implemented in the code. The pr is only to clean up the current code.
Please make a list of not needed stuff in a separate issue so we can remove them both in the strings and from the logic (code)

Reply:
Adding strings to 45 languages at once each time is a pain, so I try to add as many new strings as possible at once, including strings that may be used in the future. The only reason there are strings that are not actually used now is that each time I add a string, I add strings that I think will be needed in the future, but it takes time to develop those features.

Reply:
You should only add it to the source language (app_en.arb) and then upload it to the translation tool (poeditor currently) every translation should only be added/altered on the translation platform and not in the code.

Reply:
I know, but POEditor has a stupid behavior where when exporting translated results to GitHub, it creates 45 commits instead of one. Although I can rebase it locally into a single commit, GitHub Action makes 45 builds, and I don't want to waste resources, so I need to download each file and replace it manually.